.github/workflows/translate.yml

```diff
@@ -0,0 +1,62 @@
+name: "Update translations"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
+    
+permissions: {}
+
+defaults:
+  run:
+    shell: pwsh
+
+jobs:
+  update-translations:
+    permissions:
+      # for git push
+      contents: write
+    runs-on: ubuntu-latest
+
+    env:
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+          ref: ${{ github.ref }}
```

Comment:
Please hard-code the branch, as discussed.

Reply:
done!